packages/SwingSet/src/kernel/vatManager/manager-helper.js

```diff
@@ -139,16 +139,19 @@ function makeManagerKit(
   /**
    *
    * @param { VatDeliveryObject } delivery
-   * @returns { Promise<VatDeliveryResult> }
+   * @returns { Promise<VatDeliveryResult> } // or Error
```

Comment:
What does `or Error` mean? It can't throw or return any synchronous value since it's an `async function`?

Reply:
I'm not sure; @warner added it... I squinted at it a little bit but then moved on.